lib/diagnostic.js

```diff
@@ -5,17 +5,27 @@
 // the minimum value in an array
 // this callback will be passed by the tests to the reduce method, e.g.
 // `let minimum = array.reduce(min);`
-const min = function (previous, current) {
-
+const min = function (previous, originalArray) {
```

Comment:
Nice, this is how you wold do it with a `for loop`. However, we're looking for a function that will be passed to a reduce function